extensions/interactions/MusicNotesInput/static/css/musicNotesInput.css

```diff
@@ -8,6 +8,8 @@
   margin: 0 0 0 11.0784%;
   width: 87.34525%;
   height: 43%;
+  /*display: none; /* Hide valid note area until page loads */
```

Comment:
Ah yes! I thought the comment would be helpful toknow why it not visible initially but I guess it doesn't have to be there. Removed.